tests/jobs/rft/test_trajectory.py

```diff
@@ -93,7 +97,7 @@ def initdir(tmpdir):
 @pytest.mark.parametrize(
     "fname", ["valid_trajectories.txt", "valid_trajectories_with_comments.txt"]
 )
-def test_load(initdir, fname):
+def test_load_and_to_dataframe(initdir, fname):
```

Comment:
This test currently asserts alot of different things, is it possible to split it more into unit tests? That makes it much easier to read, and also to debug if something should start failing. Related to my comment above about helper functions.

Reply:
Split into a separate test for the dataframe structure only.